src/renderer/features/proxies/lib/proxies-utils.ts

```diff
@@ -42,7 +42,12 @@ function getNotification({
       proxyWalletName: accountsWalletsMap[proxied.proxyAccountId].name,
       proxyWalletType: accountsWalletsMap[proxied.proxyAccountId].type,
       proxiedAccountId: proxied.accountId,
-      proxiedWalletName: proxyUtils.getProxiedName(proxied.accountId, proxied.proxyType, addressPrefix),
+      proxiedWalletName: proxyUtils.getProxiedName(
```

Comment:
For  `proxyUtils.getProxiedName()` If we change first 3 arguments to a single one - `account: ProxiedAccount`
wouldn't that be more convenient? Or at least `account: Pick<ProxiedAccount, 'accountId' | 'proxyType' | 'proxyVariant'>`